PhysicsTools/NanoAOD/python/photons_cff.py

```diff
@@ -184,8 +183,13 @@ def make_bitmapVID_docstring(id_modules_working_points_pset):
         jetIdx = Var("?hasUserCand('jet')?userCand('jet').key():-1", int, doc="index of the associated jet (-1 if none)"),
```

Comment:
to save little more space: CandVars can be moved to P3vars
* int charge (is undefined for photons) 
* float mass (is trivially zero)
* pdgId from RECO code should be always 22 and not sure is of any use